catroid/src/androidTest/java/org/catrobat/catroid/test/content/controller/SceneControllerTest.java

```diff
@@ -240,6 +243,9 @@ private void createProject() throws IOException {
 		StartScript script = new StartScript();
 		PlaceAtBrick placeAtBrick = new PlaceAtBrick(0, 0);
 		script.addBrick(placeAtBrick);
+		script.addBrick(new HideTextBrick());
+		script.addBrick(new AddItemToUserListBrick());
+		script.addBrick(new AssertUserListsBrick());
```

Comment:
You wrote in the description of the PR:
> Added the 3 different types of user defined bricks leading to this problem to the scenecontroller test.

But these three are not user defined *bricks*, no? I don't understand.

Reply:
Sorry I wrote that wrong. I meant a userlist, a uservariable and a userdata type brick.